### PR TITLE
feat: update containerd to 1.6.18

### PR DESCRIPTION
**Pull request by smira**
Fixes 2 CVEs

See https://github.com/containerd/containerd/releases/tag/v1.6.18

### CODE REVIEW

File: Pkgfile [hunk 1]
```diff
@@ -7,14 +7,14 @@ vars:
 
   # renovate: datasource=github-releases depName=containernetworking/plugins
   cni_version: v1.2.0
-  cni_sha256: f3496ddda9c7770a0b695b67ae7ee80a4ee331ac2745af4830054b81627f79b7 
+  cni_sha256: f3496ddda9c7770a0b695b67ae7ee80a4ee331ac2745af4830054b81627f79b7
```
Comment (frezbo) [inline Pkgfile:10]: flannel issue :sweat_smile: ?

**Review by frezbo — APPROVED** [review]

File: Pkgfile (hunk 1 above)
Comment (smira) [reply Pkgfile:10]: just removing whitespace ;)

Comment (frezbo) [reply Pkgfile:10]: ahh nevermind, missed the whitespace

**Pull request merged**